client/src/components/main-page/posts/item/index.tsx

```diff
@@ -24,18 +25,14 @@ export const Item: FC<IProps> = ({ data }) => {
       <div className={ s.wrapper }>
          <div className={ s.top }>
             <div className={ s.data }>{ data.date }</div>
-            <div onClick={ deleteHandler } className={ s.delete }></div>
+            <div onClick={ deleteHandler } className={ s.delete }/>
          </div>
          <div className={ s.content }>
             <p className={ s.text }>{ data.text }</p>
-            { data.img &&
-               <div className={ s.imageContainer }>
-                  <img src={ `${Urls.cloudinary_url}${data.img}` } className={ s.image } alt="post" />
-               </div>
-            }
+            { data.img && <Image image={ data.img }/> }
          </div>
          <div className={ s.bottom }>
-            <Like data={ data } />
+            <Heart data={ data } />
          </div>
       </div>
    );
```

Comment:
как на счет деструктуризации?

```suggestion
            const {text, img } = data;
```

Reply:
Так мы же это как пропс прокидываем всю дату, или как ты хочешь?

Reply:
там и всю дату прокидываем и используем отдельные ее части, поэтому фиг знает, какой тут бест практис